data/rubyconf/rubyconf-2012/videos.yml

```diff
@@ -135,7 +135,7 @@
   video_id: "l07J_5luC3M"
 
 - title: "Intimate Chat with Matz"
-  raw_title: "Ruby Conf 12 - Intimate Chat with Matz featuring Yukihrio 'Matz' Matsumoto with Evan Phoenix"
```

Comment:
This speaker page should be deleted: https://www.rubyevents.org/speakers/yukihrio-matz-matsumoto

Reply:
Just deleted it 👍🏼 Thanks!